pages/services/cassandra/2.6.0-3.11.4/configuration/index.md

```diff
@@ -281,3 +281,41 @@ deploy (IN_PROGRESS)
 ### Test your multi-datacenter configuration
 
 Make sure to test your deployment using a {{ model.techShortName }} client.
+
+## Using Volume Profiles
+
+Volume profiles are used to classify volumes. For example, users can group SSDs into a “fast” profile and group HDDs into a “slow” profile. **Note:** Volume profiles are immutable and therefore cannot contain references to specific devices, nodes or other ephemeral identifiers.
+
+DC/OS Storage Service (DSS) is a service that manages volumes, volume profiles, volume providers, and storage devices in a DC/OS cluster.
+
+If you want to deploy Cassandra with DSS, please follow the tutorial here: https://docs.mesosphere.com/services/storage/1.0.0/tutorials/cassandra-dss-volumes/
+
+Once the DC/OS cluster is running and volume profiles are created, you can deploy Cassandra with the following configs:
+
+```bash
+cat > cassandra-options.json <<EOF
+{
+    "service": {
+        "volume_profile": "cassandra",
+        "disk_type": "MOUNT"
+    }
+}
+EOF
+dcos package install cassandra --options=cassandra-options.json
+
+**Note:** Cassandra will be configured to look for `MOUNT` volumes with the profile `cassandra`.
+
+Once the Cassandra service is deployed, its tasks will be running with the specified volume profiles.
+its deployment should be completed.
+
+```bash
+dcos cassandra update status
+```
+```
+deploy (serial strategy) (COMPLETE)
+└─ node-deploy (serial strategy) (COMPLETE)
+├─ node-0:[server] (COMPLETE)
+├─ node-0:[init_system_keyspaces] (COMPLETE)
+├─ node-1:[server] (COMPLETE)
+└─ node-2:[server] (COMPLETE)
```

Comment:
Lines 316-320 are not correctly formatted.

Reply:
formatted them correctly.